### DISCUSSION TITLE
Error 502 from IDP

### DISCUSSION THREAD

**Discussion by chilleregeravi** [minio/minio#17712] [Q&A]
Hi All

I have configured Azure AD Integration with Minio and get a 502 error while logging in

![image](https://github.com/minio/minio/assets/640369/2797a04a-0e1c-4ea4-b69f-99ee61c22d8f)

I have no error logs in Minio but ingress controller gives error

![image](https://github.com/minio/minio/assets/640369/3e16d92a-4220-4681-8bc9-1fea51c2bf81)

Has anyone seen this?

**Comment by vadmeste** [comment]
@chilleregeravi what version of MinIO is this ?

**Reply by chilleregeravi** [reply]
Hi @vadmeste - Release is RELEASE.2023-07-11T21-29-34Z

**Comment by chilleregeravi** [comment]
![image](https://github.com/minio/minio/assets/640369/7a8722a7-1565-488f-9efb-952ccaf77fab)
A Post request to API fails

**Reply by vadmeste** [reply]
@chilleregeravi does the cluster work fine before applying the AD change? the nginx is probably indicating that there is a crash somewhere in MinIO, any interesting logs from MinIO ?

**Reply by chilleregeravi** [reply]
Unfortunately there are no logs in minio, looks like the request doesn't reach minio.

Some things to note:
- I am using my own custom domain for console. I did find out that I should also set MINIO_SERVER_URL=console  domain name. 
- And yes, the console login does work with the static user credentials. Only in the case of OIDC is the call to api/v1/login failing. 
- I have set the redirect url to the console url name in both the App registration in Azure and also in MINIO
